Hello Microverse Project Setup

maximoortelli opened the pull request:
In this version the files were added: index.html, Style.css, .gitignore, Readme.md, cat-black-face, LICENSE, /.github/workflows/linters

maximoortelli committed:
Add New Files

In this version the files were added: index.html, Style.css, .gitignore, Readme.md, cat-black-face, LICENSE, /.github/workflows/linters

binhussen commented on README.md:
```diff
@@ -0,0 +1,170 @@
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+### No Live demo available.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
- Great for describing as it hasn’t the live version, I think it would be better if you remove this section at all to make your readme more professional 👌

maximoortelli replied:
Done

binhussen reviewed (changes requested):
Hi @maximoortelli , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media0.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>

## Highlights

- Describtive commit ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
- Kindly add the `gitignore` file to prevent unnecessary files from your repository 👌
- Kindly add the `test.md` file and ignore it by gitigonre as it is part of the [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)
- Kindly set up the [ linter](https://github.com/microverseinc/linters-config) to make your code clean and follow best practices 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

maximoortelli committed:
Delete LICENSE.txt

maximoortelli committed:
Hello-Microverse project update

Implementations in this pull request;

In this new version I added the .gitignore file, in it contains test.md and node_modules.

In the README.md file, remove the live version section to make my readme more professional.

levy002 commented on README.md:
````diff
@@ -0,0 +1,161 @@
+4. Open Terminal.
+5. Change the current working directory to the location where you want the cloned directory.
+6. Type git clone, and then paste the URL you of the reposotory.<br>
+``` $ git clone https://github.com/YOUR-USERNAME/YOUR-REPOSITORY ```
````
- [x] Kindly make sure to remove this placeholder and add your repository link for cloning purposes.

maximoortelli replied:
Thank you very much and made the changes

levy002 reviewed (changes requested):
## Hello @maximoortelli     , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif)

## Highlights
- Correct Git-flow :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [x] Kindly make sure to update your PR title by proving a well-detailed and self-explanatory title that makes your PR clean and professional. :wink: 

    - Example of a good PR title
    _**Hello Microverse Project Setup**_

   ### HOW TO EDIT PR TITLE
    ![chrome-capture-2023-3-27 (1)](https://user-images.githubusercontent.com/87197412/234717588-992cb1ef-8a21-4987-ac09-5c833bfabc41.gif)


- [x] Kindly make sure to set up linters correctly so it runs to check if there are no errors on your pull request. I recommend you take a step back and read this[ lesson](https://github.com/microverseinc/linters-config/blob/master/html-css) to understand it clearly and follow the guideline to set it correctly.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

maximoortelli committed:
Hello Microverse Project Setup Update

Changes were made to the README.md file. 
Where was the link of this GITHUB repository placed.

levy002 commented:
Hello @maximoortelli. If you have implemented the suggested changes, kindly go ahead and submit your PR link again in the dashboard in order to get feedback. Keep rocking mate :rocket:

Qoosim reviewed (changes requested):
#  *Status: Required Changes* :recycle: 

Hi @maximoortelli, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 

![](https://camo.githubusercontent.com/38ce8c8b213b08e2d9bb85a0b79702ad512f3910f803ec3d7917b4e0110c9044/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966)

 ## Required Changes ♻️
_Check the comments under the review._

- [x] As suggested by the previous reviewer, kindly make sure to set up linters correctly so it runs to check if there are no errors on your pull request. I recommend you take a step back and read this[ lesson](https://github.com/microverseinc/linters-config/blob/master/html-css) to understand it clearly and follow the guideline to set it up correctly.

## Optional suggestions

- [OPTIONAL] Kindly set up MIT License for your project. Currently, the link is broken. You can follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to properly set it for your project.

![license](https://user-images.githubusercontent.com/26597432/235531825-02ca4fd8-2d59-4851-aa78-d3e68d95c9ec.png)
 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

maximoortelli committed:
Add the linters Configuration

maximoortelli committed:
Add gitignore file

maximoortelli committed:
Delete LICENSE

maximoortelli committed:
I add the license

I add the MIT license to the branch

maximoortelli committed:
Add Linters

The linters are now configured correctly.

maximoortelli commented:
Thank you, all the errors mentioned in the comments have already been corrected.

maximoortelli commented:
Thank you, all the errors mentioned in the comments have already been corrected.

victorokoroji reviewed (approved):
Hi @maximoortelli , 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested change implementated ✅ 

<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
